suite/src/__tests__/fast/model-correctness.test.ts

```diff
@@ -0,0 +1,57 @@
+import { describe, test, beforeAll, expect } from '@jest/globals'
+import { newCeramic } from '../../utils/ceramicHelpers.js'
+import { createDid, splitDidsToList } from '../../utils/didHelper.js'
+import { StreamID } from '@ceramicnetwork/streamid'
+import {Model} from '@ceramicnetwork/stream-model'
+import {ModelInstanceDocument} from '@ceramicnetwork/stream-model-instance'
+import { newModel, basicModelDocumentContent } from '../../models/modelConstants'
+import { CeramicClient } from '@ceramicnetwork/http-client'
+import { CommonTestUtils as TestUtils } from '@ceramicnetwork/common-test-utils'
+
+const ComposeDbUrls = String(process.env.COMPOSEDB_URLS).split(',')
+// TODO : Update to the modelId created manually on all envs, if id cannot be the same then create an array for different envs  
+let modelId = "kjzl6hvfrbw6c59m8k5l50gqmsduszfazp25wakaep3nsiyieff7dbdvpgb0ypf"
+let seed = String(process.env.COMPOSEDB_ADMIN_DID_SEEDS)
```

Comment:
`COMPOSEDB_ADMIN_DID_SEEDS` will also be a comma-separated list like `COMPOSEDB_URLS`. Can we split it here instead of calling `splitDidsToList`?
```suggestion
const AdminDidSeeds = String(process.env.COMPOSEDB_ADMIN_DID_SEEDS).split(',')
```